mono/mini/mini-exceptions.c

```diff
@@ -699,11 +699,11 @@ unwinder_unwind_frame (Unwinder *unwinder,
 
 		/* Process debugger invokes */
 		/* The DEBUGGER_INVOKE should be returned before the first interpreter frame for the invoke */
-		if (unwinder->last_frame_addr > (gpointer)(*lmf)) {
-			if (((guint64)(*lmf)->previous_lmf) & 2) {
+		if (unwinder->last_frame_addr > (gsize)(*lmf)) {
```

Comment:
I don't understand this change: `Unwinder:last_frame_addr` is a `gpointer`, why do you need to cast `*lmf` to `gsize`?

Reply:
thanks, this is wrong and actually introduced a new warning 😅

Reply:
No need for the parens around *lmf. There is no ambiguity when you have prefix-oerator identifier right-paren. Extra parens are noisy and do not disambiguate, despite what some think.

if (a == *b) // no ambiguity -- this cannot read "(a == *)b" or "a (== *) b"
if (a == (*b)) // but people write this